[2022-10-26 13:38:00,033] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.resultados_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 13:38:00,097] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.resultados_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 13:38:00,098] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 13:38:00,098] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-10-26 13:38:00,098] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 13:38:00,173] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): resultados_raw_populate> on 2022-09-01 00:00:00+00:00
[2022-10-26 13:38:00,180] {standard_task_runner.py:52} INFO - Started process 483 to run task
[2022-10-26 13:38:00,186] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_raw_data', 'resultados_raw_populate', 'scheduled__2022-09-01T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/raw-dag.py', '--cfg-path', '/tmp/tmp156_xus_', '--error-file', '/tmp/tmpfh5iwpej']
[2022-10-26 13:38:00,188] {standard_task_runner.py:80} INFO - Job 24: Subtask resultados_raw_populate
[2022-10-26 13:38:00,339] {task_command.py:369} INFO - Running <TaskInstance: load_raw_data.resultados_raw_populate scheduled__2022-09-01T00:00:00+00:00 [running]> on host b1681a16582a
[2022-10-26 13:38:00,549] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_raw_data
AIRFLOW_CTX_TASK_ID=resultados_raw_populate
AIRFLOW_CTX_EXECUTION_DATE=2022-09-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-01T00:00:00+00:00
[2022-10-26 13:38:00,571] {base.py:68} INFO - Using connection ID 'postgres-default' for task execution.
[2022-10-26 13:38:00,578] {dbapi.py:208} INFO - Running statement: COPY resultados_raw FROM '/tmp/sample_data/resultados.csv' DELIMITER ',' CSV HEADER;, parameters: None
[2022-10-26 13:38:08,365] {dbapi.py:216} INFO - Rows affected: 270907
[2022-10-26 13:38:08,380] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=load_raw_data, task_id=resultados_raw_populate, execution_date=20220901T000000, start_date=20221026T133800, end_date=20221026T133808
[2022-10-26 13:38:08,404] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-26 13:38:08,437] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-10-26 14:27:40,837] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.resultados_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 14:27:40,853] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.resultados_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 14:27:40,853] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 14:27:40,853] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-10-26 14:27:40,853] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 14:27:40,874] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): resultados_raw_populate> on 2022-09-01 00:00:00+00:00
[2022-10-26 14:27:40,878] {standard_task_runner.py:52} INFO - Started process 273 to run task
[2022-10-26 14:27:40,881] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_raw_data', 'resultados_raw_populate', 'scheduled__2022-09-01T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/raw-dag.py', '--cfg-path', '/tmp/tmp711n8l_a', '--error-file', '/tmp/tmp9ggirwr9']
[2022-10-26 14:27:40,882] {standard_task_runner.py:80} INFO - Job 16: Subtask resultados_raw_populate
[2022-10-26 14:27:40,946] {task_command.py:369} INFO - Running <TaskInstance: load_raw_data.resultados_raw_populate scheduled__2022-09-01T00:00:00+00:00 [running]> on host d860ef10ab5f
[2022-10-26 14:27:41,038] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_raw_data
AIRFLOW_CTX_TASK_ID=resultados_raw_populate
AIRFLOW_CTX_EXECUTION_DATE=2022-09-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-01T00:00:00+00:00
[2022-10-26 14:27:41,053] {base.py:68} INFO - Using connection ID 'postgres-default' for task execution.
[2022-10-26 14:27:41,058] {dbapi.py:208} INFO - Running statement: COPY resultados_raw FROM '/tmp/sample_data/resultados.csv' DELIMITER ',' CSV HEADER;, parameters: None
[2022-10-26 14:27:47,993] {dbapi.py:216} INFO - Rows affected: 270907
[2022-10-26 14:27:48,008] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=load_raw_data, task_id=resultados_raw_populate, execution_date=20220901T000000, start_date=20221026T142740, end_date=20221026T142748
[2022-10-26 14:27:48,054] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-26 14:27:48,087] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-10-26 14:47:54,521] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.resultados_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 14:47:54,537] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.resultados_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 14:47:54,537] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 14:47:54,537] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-10-26 14:47:54,537] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 14:47:54,556] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): resultados_raw_populate> on 2022-09-01 00:00:00+00:00
[2022-10-26 14:47:54,560] {standard_task_runner.py:52} INFO - Started process 312 to run task
[2022-10-26 14:47:54,564] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_raw_data', 'resultados_raw_populate', 'scheduled__2022-09-01T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/raw-dag.py', '--cfg-path', '/tmp/tmpux65siup', '--error-file', '/tmp/tmp3b08_31p']
[2022-10-26 14:47:54,565] {standard_task_runner.py:80} INFO - Job 16: Subtask resultados_raw_populate
[2022-10-26 14:47:54,632] {task_command.py:369} INFO - Running <TaskInstance: load_raw_data.resultados_raw_populate scheduled__2022-09-01T00:00:00+00:00 [running]> on host 4af2e895db31
[2022-10-26 14:47:54,732] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_raw_data
AIRFLOW_CTX_TASK_ID=resultados_raw_populate
AIRFLOW_CTX_EXECUTION_DATE=2022-09-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-01T00:00:00+00:00
[2022-10-26 14:47:54,744] {base.py:68} INFO - Using connection ID 'postgres-default' for task execution.
[2022-10-26 14:47:54,749] {dbapi.py:208} INFO - Running statement: COPY resultados_raw FROM '/tmp/sample_data/resultados.csv' DELIMITER ',' CSV HEADER;, parameters: None
[2022-10-26 14:48:01,965] {dbapi.py:216} INFO - Rows affected: 270907
[2022-10-26 14:48:01,980] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=load_raw_data, task_id=resultados_raw_populate, execution_date=20220901T000000, start_date=20221026T144754, end_date=20221026T144801
[2022-10-26 14:48:02,020] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-26 14:48:02,075] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
